src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -48,7 +48,7 @@ public CommentAttributes(String courseId, String giverEmail, CommentParticipantT
         this.giverEmail = giverEmail;
         this.recipientType = recipientType == null ? CommentParticipantType.PERSON : recipientType;
         this.recipients = recipients;
-        this.commentText = commentText;
+        this.commentText = commentText == null ? null : new Text(Sanitizer.sanitizeForRichText(commentText.getValue()));
```

Comment:
As it's used in other places unrelated to this issue, it'd make sense to do it in a separate issue